test/deconvolution-operator-tester.h

```diff
@@ -1148,7 +1148,7 @@ class DeconvolutionOperatorTester {
       for (size_t g = 0; g < groups(); g++) {
         for (size_t oc = 0; oc < group_output_channels(); oc++) {
           int32_t range = w8dist(rng);
-          auto weights_dist = std::uniform_int_distribution<int32_t>(std::min(range, 0), std::max(range, 0));
+          auto weights_dist = std::uniform_int_distribution<int32_t>(std::min<int32_t>(range, 0), std::max<int32_t>(range, 0));
```

Comment:
INT32_C(0) for the constants instead?

Reply:
Quickly tried INT32_C(0) but no luck unfortunately. I will keep investigating if we have better solution than this.